src/calendar/utils/__tests__/navigation.test.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { moveNextDay, movePrevDay, moveNextWeek, movePrevWeek, getBaseDate } from '../navigation';
+
+jest.mock('date-fns', () => ({ ...jest.requireActual('date-fns'), startOfMonth: () => new Date('2022-01-01') }));
```

Comment:
How about using `mockdate` here?

Reply:
I am not familiar with this one. Does it handle timezones?

The reason I am mocking the `startOfMonth` here is to avoid the UTC `2020-01-01` date to be converted to local as it requires tricky assertions in that case.

Reply:
Looks like it does not.

That's unfortunate, mocking stateless utility libraries (like date-fns) is a bad practice, but I have no alternative. Let's merge it